apps/web/src/components/UserProfileDocument/UserProfileDocument.tsx

```diff
@@ -210,10 +192,10 @@ const UserProfileDocument = React.forwardRef<
                           style={{ flex: "1 1 0%" }}
                         >
                           {intl.formatMessage({
-                            defaultMessage: "Employment Equity Information",
-                            id: "aa7B7S",
+                            defaultMessage: "Employment equity information",
```

Comment:
Ya, I wanted to do this but it is a different string, **Diversity, equity and inclusion**. Maybe it can be changed.

@Jerryescandon would it be acceptable to change the heading on the print/save pdf feature from **Employment equity information** to **Diversity, equity and inclusion**?

Reply:
Yeah do it.